### PR TITLE
 Issue 4 fix: TJVendors Reports and Profile page is looking messed up for BS2 Template

### PR DESCRIPTION
**Pull request by deepa-g**

### CODE REVIEW

File: src/com_tjvendors/site/views/vendor/tmpl/default.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
 			?>
 	</h1>
 	<input type="hidden" name="client" value="<?php echo $this->input->get('client', '', 'STRING'); ?>" />
-	<div class="profile row" id="tjv-wrapper">
+	<div class="profile row tjBs3" id="tjv-wrapper">
```
Comment (manojLondhe) [inline src/com_tjvendors/site/views/vendor/tmpl/default.php]: declare a constant for classname in entry file, use constant in views

File: src/com_tjvendors/site/tjvendors.php [hunk 2]
```diff
@@ -33,3 +34,13 @@
 // Frontend css
 JHtml::_('stylesheet', 'com_tjvendor/tjvendors.css', $options);
 JHtml::stylesheet(JUri::root() . 'media/techjoomla_strapper/vendors/no-more-tables.css', array(), true);
+
+// Load Boostrap
+$params         = ComponentHelper::getParams('com_tjvendor');
```
Comment (manojLondhe) [inline src/com_tjvendors/site/tjvendors.php]: isn't it `com_tjvendors` and not `com_tjvendor` ?

Comment (deepa-g) [reply src/com_tjvendors/site/tjvendors.php]: Updated

File: src/com_tjvendors/site/views/vendor/tmpl/default.php (hunk 1 above)
Comment (deepa-g) [reply src/com_tjvendors/site/views/vendor/tmpl/default.php]: Updated everywhere

File: src/com_tjvendors/site/defines.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @version    SVN:<SVN_ID>
+ * @package    SocialAds
```
Comment (manojLondhe) [inline src/com_tjvendors/site/defines.php]: change name, copyright year

File: src/com_tjvendors/site/tjvendors.php [hunk 4]
```diff
@@ -8,11 +8,16 @@
...
 defined('_JEXEC') or die;
+use Joomla\CMS\Component\ComponentHelper;
 
 // Include dependancies
 jimport('joomla.application.component.controller');
 
 JLoader::registerPrefix('Tjvendors', JPATH_COMPONENT);
+
+// Load defines.php
+require_once JPATH_SITE . '/components/com_tjvendors/defines.php';
```
Comment (manojLondhe) [inline src/com_tjvendors/site/tjvendors.php]: remove

**Pull request merged**